src/commands/jobs/pause.ts

```diff
@@ -1,19 +1,67 @@
 import { runCommand } from '../../util/command';
-import { detailOutputFormat } from './util';
+import { detailOutputFormat, jobsSearchArgumentsDefault, searchCriteriaIsMissing } from './util';
 
 export = {
-	command: 'pause <id>',
-	desc: 'Pause job',
+	command: 'pause [id]',
+	desc: 'Pause job(s)',
+	builder: yargs => {
+		yargs.options({
+			'all': {
```

Comment:
I don't think this is a valid parameter.

Reply:
@rklind and I were talking about forcing the user to specify --all to start/pause all jobs as a safety precaution. Should I remove that safety check, or maybe implement it a different way?

Reply:
That's fine but you pass it down to the parameters block that is serialized to the service. We shouldn't do that.

Reply:
Ah, gotcha

Reply:
What if rather than using `--all` option, it keyed off of the `id` positional argument? That would allow you to keep `pause <id>` so that the positional argument is required so that you don't have to check if it is available below. You would simply check `argv.id === 'all'`. This would change the usage from `skysync-cli start --all` to `skysync-cli start all`. Thoughts?

Reply:
It's possible to call start/pause without `id` or `all`.
For example, `skysync-cli jobs start --parent=1234`
I'm leaning towards leaving the `--all` switch as-is, but I'm open to discussing this further